overlays/holo-nixpkgs/hpos-led-manager/src/main.rs

```diff
@@ -53,17 +86,53 @@ fn main() -> Fallible<()> {
 
         let hpos_config_found = Path::new("/run/hpos-init/hpos-config.json").exists();
 
-        let state = match (online, hpos_config_found) {
-            (false, _) => State::Flash(Color::Purple),
-            (true, false) => State::Static(Color::Blue),
+        if counter % HYDRA_POLLING_INTERVAL == 0 {  
+            revision_json = get_hydra_revision()?;
```

Comment:
This means we should `await` this now.

```suggestion
            revision_json = get_hydra_revision().await?;
```